### PR TITLE
Add chartbeat analytics component to homepage, with unit and integration tests

### PR DESCRIPTION
**Pull request by LilyL0u**
Chartbeat analytics component is added to the HomePage component. Unit test checks if the chartbeat analytics component is in the document. Integration tests now have amp analytics tests running on the home page.

Testing
======

Manual testing steps are on the corresponding JIRA ticket. Unit and integration tests pass.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

**Comment by LilyL0u** [comment]
Screenshot 2023-05-22 at 13 52 28


There are a few differences in the request headers there at the bottom e.g Sec-Fetch-Mode. Is this just live vs localhost differences or something we need to implement at some point?

**Review by shayneahchoon — APPROVED** [review]

**Review by louisearchibald — APPROVED** [review]

File: src/app/pages/HomePage/index.test.tsx [hunk 1]
```diff
@@ -4,6 +4,12 @@ import { Helmet } from 'react-helmet';
 import { render } from '../../components/react-testing-library-with-providers';
 import HomePage from './HomePage';
 
+jest.mock('../../components/ThemeProvider');
```
Comment (karinathomasbbc) [inline src/app/pages/HomePage/index.test.tsx]: Do we need to mock the theme provider here?

**Review by karinathomasbbc — APPROVED** [review]
Niiiice, thanks for this 🍭

**Comment by karinathomasbbc** [comment]
> There are a few differences in the request headers there at the bottom e.g Sec-Fetch-Mode. Is this just live vs localhost differences or something we need to implement at some point?

You're right, the request headers are slightly different on live vs localhost, and we don't need to worry about them 🙌

File: src/app/pages/HomePage/index.test.tsx (hunk 1 above)
Comment (LilyL0u) [reply src/app/pages/HomePage/index.test.tsx]: I'll remove it and see if it affects anything

Comment (LilyL0u) [reply src/app/pages/HomePage/index.test.tsx]: it did not. I have removed it now.

**Pull request merged**